### DISCUSSION TITLE
What is the impact of sap-netweaver-info-leak.yaml?

### DISCUSSION THREAD

**Discussion by SAGEof6iixPATHS** [projectdiscovery/nuclei-templates#1736] [Q&A] [answered]
Hey!

I caught this in one of the websites
[sap-netweaver-info-leak.yaml](https://github.com/projectdiscovery/nuclei-templates/blob/0013f94807c721313535e132b0aa97b368c7e18b/misconfiguration/sap/sap-netweaver-info-leak.yaml)

I am not able to find anything on the web to report... Is it worth to report? If yes then what to report??

**Comment by ehsandeep** [comment, marked as answer]
@SAGEof6iixPATHS feel free to check those references - https://github.com/projectdiscovery/nuclei-templates/commit/49f9b6782720f00eb86fa9fed1c4d28c31c449d6
